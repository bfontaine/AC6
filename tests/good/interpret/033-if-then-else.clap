val x = 0
val z = if x = 0 then 1 else 2

