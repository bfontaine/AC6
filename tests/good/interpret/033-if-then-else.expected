:- x = 0
:- z = 1
